apps/website/src/pages/show-and-tell/index.tsx

```diff
@@ -315,6 +315,7 @@ const ShowAndTellIndexPage: NextPage<ShowAndTellPageProps> = ({
       : lastPageItems?.[lastPageItems.length - 1]?.id;
   }, [entries]);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies:
```

Comment:
Why deps are missing?

Reply:
I added the ignore comment before I decided to downgrade the rule to just warn

Reply:
Less worried about the level of the rule, was more just curious what is actually missing, and if it should be fixed rather than marked as an ignore?

Reply:
I think the `entries.data` is unnecessary according to biome.
`⚠ This hook specifies more dependencies than necessary: entries.data`

Reply:
I don't know if it can be safely removed

Reply:
@pjeweb 👀 ?

Reply:
No, it cant be removed. There is probably a better way to do this, but my brain aint braining rn. What I am doing there is running the effect to scroll to the newest element after data has changed and has been rendered. The dependency is not used in the effect body, but its used to only trigger when the data actually changes.